### PR TITLE
Fix empty native lib path in Flutter app

### PR DESCRIPTION
**Pull request by d-uzlov**
Closes https://github.com/mlcommons/mobile_app_open/issues/155

### CODE REVIEW

**Review by anhappdev — COMMENTED** [review]
The code looks good to me. Though, I have no suitable device to validate if the vendor backend works correctly.

File: flutter/lib/backend/native.dart [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 import 'package:device_info/device_info.dart';
 import 'package:ffi/ffi.dart';
 
 import 'package:mlcommons_ios_app/protos/backend_setting.pb.dart' as pb;
 import 'package:mlcommons_ios_app/protos/mlperf_task.pb.dart' as pb;
 import 'bridge.dart';
 
+const androidChannel = MethodChannel('org.mlcommons.mlperfbench/android');
+Future<String> getNativeLibraryPath() async {
+  return await androidChannel.invokeMethod('getNativeLibraryPath') as String;
```
Comment (mohitmundhragithub) [inline flutter/lib/backend/native.dart:15]: will this work for other OS as well, like windows / iOS?

Comment (d-uzlov) [reply flutter/lib/backend/native.dart:15]: This method will throw an exception if we try to run it on Windows/iOS.
We only call it if `Platform.isAndroid` returned true:
https://github.com/mlcommons/mobile_app_open/blob/dea36c31605311a4b98d9d1f43146e479ace4dbd/flutter/lib/benchmark/benchmark.dart#L630-L631

Comment (d-uzlov) [reply flutter/lib/backend/native.dart:15]: I don't know of any equivalent of `nativeLibraryDir` for other platforms.
On iOS we don't use any custom dynamic libraries yet. And we only use tflite backend, and there are no plans on adding more backends, as far as I'm aware.
On Windows the package is simpler that on Android/iOS, and we just keep all `.dll` files near executable.

**Comment by d-uzlov** [comment]
@mohitmundhragithub, do you have any more comments, question or requests regarding this change?
If not, could you approve this PR?

**Review by mohitmundhragithub — APPROVED** [review]

**Pull request merged**